src/openrct2/CmdlineSprite.cpp

```diff
@@ -819,7 +819,7 @@ int32_t cmdline_for_sprite(const char** argv, int32_t argc)
         }
 
         json_decref(sprite_list);
-        free(directoryPath);
+        delete directoryPath;
```

Comment:
Keep this PR just for the easy self contained changes leave this for another PR.